src/index.ts

```diff
@@ -744,11 +790,13 @@ async function distributeFnTransactional(
       // Write to firestore in one transaction
     for (const [_, logicDocs] of transactionalDstPathLogicDocsMap) {
       for (const logicDoc of logicDocs) {
-        await distributeDoc(logicDoc, undefined, txn);
+        logicDocsForQueuing.push(logicDoc);
```

Comment:
No need to collect logicDocs for queueing since we modified the doc of original logicDoc.  It should now be included in the logicResults

Reply:
You should not remove distributeDoc because the transactional doc will not be written in the database